src/twisted/conch/ssh/keys.py

```diff
@@ -289,6 +304,9 @@ def _fromString_PRIVATE_BLOB(cls, blob):
         elif keyType == b'ssh-dss':
             p, q, g, y, x, rest = common.getMP(rest, 5)
             return cls._fromDSAComponents(y=y, g=g, p=p, q=q, x=x)
+        elif keyType == b'ssh-ed25519':
+            pr = nacl.signing.SigningKey(common.getNS(rest)[0])
```

Comment:
The variable name `pr` has me a bit confused; can we pick something more intuitive?